app/views/collection_numbers/_form.html.erb

```diff
@@ -1,7 +1,6 @@
-<%= # form_tag(add_query_param(action: :new, id: @observation.id)) do %>
-<%= form_with model: :collection_number do |f| %>
+<%= form_with model: @collection_number do |f| %>
 
-  <%= submit_tag(button.t, class: "btn mx-auto mt-3") %>
+  <%= submit_tag(local_assigns[:button].t, class: "btn mx-auto mt-3") %>
```

Comment:
The combination of `locals: { button: :SAVE }` in the view and `button` in the form was not working (for reasons I don't understand.